src/main/java/duke/exception/UnknownInputFieldError.java

```diff
@@ -0,0 +1,4 @@
+package duke.exception;
+
+public class UnknownInputFieldError extends Exception {
```

Comment:
since it is a custom exception, you need to provide Javadoc comments to indicate its purpose in the codebase